src/test/java/HomeWork14.java

```diff
@@ -10,18 +10,12 @@ public class HomeWork14 {
     public static void navigation() {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://testpro.io/";
+        String url = "https://bbb.testpro.io/";
         driver.get(url);
-
         WebElement registrationLink = driver.findElement(By.cssSelector("a[type = submit]"));
         registrationLink.click();
         String registrationUrl = "https://bbb.testpro.io/registration.php";
         Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
         driver.quit();
-
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210208834-427fff55-bfc2-4d4d-a6da-874bdc1fab59.png)

For example `public class HomeWork14 extends BaseTest`